### ISSUE TITLE
Windows: Works once, editor hangs second time

### ISSUE THREAD

**Issue by charliebruce** [charliebruce/cobble#6]
This appears to be because the plugin is (implicitly?) creating a thread, and Mono waits for this thread to exit before starting the next game session.

https://fogbugz.unity3d.com/default.asp?1065590_p5vr9ftohapf1apf
https://mbientlab.com/community/discussion/2601/unity-works-perfectly-with-metawear-sensor-on-first-run-hangs-on-2nd-run/p2

I'm not entirely clear how or where this thread is being created. Somewhere in the WinRT shim logic? Something about callbacks?

**Comment by charliebruce** [comment]
Seems to be working correctly as of now. Will treat any odd behaviour from here as a separate issue.
